src/components/atoms/Timeline/index.tsx

```diff
@@ -35,15 +52,76 @@ const Timeline: React.FC<Props> = memo(
       strongScaleHours,
       currentPosition,
       events,
+      player: {
+        playSpeed,
+        onChangePlaySpeed,
```

Comment:
onPlaySpeedChange

Please see [here](https://github.com/reearth/guides/blob/main/frontend/style.md#functions) for word order of functions

Reply:
This is fixed by [chore: rename to onPlaySpeedChange](https://github.com/reearth/reearth-web/pull/277/commits/8848e5506830ecceb5b4c1bc83eb4d7600d703a5)